dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -82,6 +83,16 @@ def after_last_child_position(course_version_id, parent_key)
       last&.position || 0) + 1
   end
 
+  def redirect_unit_group
+    course_name = params[:course_course_name]
+
+    # When the url of a course family is requested, redirect to a specific course version.
+    if UnitGroup.family_names.include?(course_name)
+      unit_group = UnitGroup.latest_stable_version(course_name)
+      redirect_to action: params[:action], course_course_name: unit_group.name, key: params[:key] if unit_group
```

Comment:
doing a redirect in a before-action seems a little strange, but this is the pattern used by the existing latest-version redirects like the one on `courses#standards`.